menpo/transform/test/with_dims_test.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+from menpo.shape import PointCloud
+from menpo.transform import WithDims
+
+
+def test_withdims_transform():
```

Comment:
Add a test for when you slice with a single element (which you can do - a single integer is a valid use of this function)